docs/ConfigurationFile.md

```diff
@@ -119,6 +119,23 @@ exports.config = {
     // Level of logging verbosity: trace | debug | info | warn | error
     logLevel: 'info',
     //
+    // Set specific log levels per logger
+    // use 'silent' level to disable logger
+    logLevels: {
+        webdriver: 'info',
+        'wdio-applitools-service': 'info'
+    },
+    //
+    // If outputDir is provided WebdriverIO can capture driver session logs
+    // it is possible to configure which logTypes to include/exclude.
+    driverLog: {
```

Comment:
This is problematic, if you run tests with different browser and different driver supporting different log types we run into issues. This needs to live on the capability level.